package.json

```diff
@@ -13,27 +13,14 @@
   "scripts": {
     "postinstall": "electron-builder install-app-deps && rimraf node_modules/dtrace-provider",
     "start": "electron .",
-    "asarl": "asar l release/mac/Signal.app/Contents/Resources/app.asar",
+    "grunt": "grunt",
     "icon-gen": "electron-icon-maker --input=images/icon_1024.png --output=./build",
     "generate": "npm run icon-gen && grunt",
     "build": "build  --config.extraMetadata.environment=$SIGNAL_ENV",
-    "dist": "npm run generate && npm run build",
-    "pack": "npm run generate && npm run build -- --dir",
+    "build-release": "SIGNAL_ENV=production npm run build -- --config.directories.output=release",
```

Comment:
Could we try and switch over to `yarn` vs `npm`, e.g. `yarn build` vs `npm run build` so we can move towards having the locally checked-in `yarn.js`?

Reply:
Not sure how those two are related. As I've mentioned before, there were difficulties with switching `npm` to `yarn` in all of these embedded scripts. It'll be a pretty substantial testing process to move things over.

Reply:
Got it. In my experience, changing `npm run <script>` to `yarn <script>` always seemed to have worked but I’d love to learn more about where troubles popped up.